### DISCUSSION TITLE
Password not recognized

### DISCUSSION THREAD

**Discussion by mcarmel56** [iotaledger/firefly#4058] [Help & Support]
I triple checked that I entered the correct password. Say not recognized.

**Comment by Alpenflitzer** [comment]
Ich hatte und habe immer wieder das selbe Problem. Bei beiden Wallets wird das Passwort nicht mehr erkannt (schon zum 6 mal)
muss dann immer wieder neue Wallets anlegen und mit der Wiederherstellung beginnen. Ärgerlich und unheimlich aufwendig. Gibt es das schon eine Lösung. Es betrifft anscheinend nicht sehr viele
